crates/turbo-tasks/src/lib.rs

```diff
@@ -99,7 +99,7 @@ pub use read_ref::ReadRef;
 pub use state::State;
 pub use task::{
     concrete_task_input::{ConcreteTaskInput, SharedReference, SharedValue},
-    task_input::TaskInput,
+    task_input::{RcStr, TaskInput},
```

Comment:
Can we stop using re-exports in rust?

Reply:
My opinion is that this is fine. I'm okay with re-exports at the top-level of a crate that re-export private internals of the crate, especially as rust-analyzer's "go to definition" makes this stuff easy to follow.

The problems I see with re-exports are:
- Cross-crate re-exports that bottleneck the compilation graph, e.g. `turbopack-binding`
- Re-exporting things that are already public, leading to imports from two different competing places because nobody is sure where they should import from.
- Re-exports that just lead to more confusion about the underying type, e.g. `turbo_tasks::Result` is a re-export of `anyhow::Result`, but since `anyhow::Result` is a more widely understood type it would be better to use it directly (and most code does!)